packages/manager/apps/hpc-vmware-managed-vcd/src/components/modal/EditDetailModal.tsx

```diff
@@ -113,6 +118,7 @@ export const EditDetailModal = ({
         </OsdsText>
       </OsdsFormField>
 
+      {isLoading && <Loading slot="actions" className="w-9 mr-4" />}
```

Comment:
Should not be a skeleton rather ?

Reply:
Not here, we should display a loader as it's not some text or content loading, it's simply to indicate that the form request is pending (with the "confirm button" disabled)